tests/test_surfaces.py

```
@@ -84,6 +84,7 @@ def test_surface_init(self):
 
     def test_validator(self):
         surf = Surface()
+        # TODO is IllegalState or defaults values more desirable?
```

@tjlaboss with this PR almost everything will have default values now, Cylinders default to `CZ` or `C/Z`, planes to `PZ`. Constants default to `0.0`, and the number to `-1`. So the question is is there are default values now should `IllegalState` be raised commonly if at all?

I think at least for a surface number of `-1` this should raise an error, but for most other things I'm thinking now?

---

Didn't see this comment until I left some remarks of my own in other files

Do they really need defaults? I don't think they do and that this is more likely to cause a user to have an accident